src/blocks/mrc_event_handler.ts

```diff
@@ -134,6 +135,9 @@ const EVENT_HANDLER = {
       });
     });
     this.mrcUpdateParams();
+    
+    // Update all mrc_get_parameter blocks to recheck validity
+    this.mrcCheckParameterBlocks();
```

Comment:
Is this necessary? Since mrcOnLoad calls checkEvent, which calls mrcCheckParameterBlocks, I don't think this is necessary.
Also, I'm concerned that the parameter blocks might not be fully formed at the time that this block's loadExtraState is called.

Reply:
You are right.  It isn't necessary (and could be problematic).  I removed it